src/components/FilterSidebar/components/QAFilter.tsx

```diff
@@ -0,0 +1,24 @@
+import { ChangeEvent } from 'react';
+
+import { FilterData } from '@/context/FilterContext/types';
+
+type QAFilterProps = {
+  filterData: FilterData;
+  onChange: (event: ChangeEvent<HTMLSelectElement>) => void;
+};
+
+export const QAFilter = ({ filterData, onChange }: QAFilterProps) => (
+  <div>
+    <label htmlFor="options">{filterData.name}</label>
+
+    <select id="options" onChange={onChange}>
+      <option value="none">Please select an option</option>
+
+      {filterData.options.map((option) => (
+        <option key={option.value} value={option.value}>
```

Comment:
We should also pass what value is selected, especially if we are 'storing' the filter state via URL query params in the future.

Reply:
Can you clarify what you mean by`pass what value is selected`? Pass it where?

Right now filter state is in context, it should either be in context, or in the URL, not both (in my opinion).

Reply:
For when the user flows passes through the Keycloak auth flow, we will lose the context and it will have to be repopulated.

In the events that happens, none of the `<options>` will be selected.

Reply:
I'm not going to do this right now as that should be done altogether for all filters. 

There is a ticket to reset `filters` after auth, that is the appropriate ticket for that to come under. It will mean removing the current `FilterContext` and moving any state there to the URL as a `queryString`. The client is wrapped in a `react-router-dom` Provider, so we can then use `UseSearchParams` hook wherever we currently use `useFilters`.